app/views/container_project/_main.html.haml

```diff
@@ -1,8 +1,13 @@
-= render :partial => "layouts/flash_msg"
-.row
-  .col-md-12.col-lg-6
-    = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"), :items => textual_group_properties}
-  .col-md-12.col-lg-6
-    = render :partial => "shared/summary/textual", :locals => {:title => _("Relationships"), :items => textual_group_relationships}
-    = render :partial => "shared/summary/textual_tags", :locals => {:title => _("Smart Management"), :items => textual_group_smart_management}
+- if @showtype == "timeline"
```

Comment:
this if block is not needed in here, this has been taken care of in app/views/container_project/show.html.haml below.